app/assets/javascripts/discourse/initializers/topic-footer-buttons.js.es6

```diff
@@ -0,0 +1,116 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "topic-footer-actions",
+
+  initialize() {
+    withPluginApi("0.8", api => {
+      api.registerTopicFooterButton({
+        id: "share",
+        icon: "link",
+        priority: 999,
+        label: "topic.share.title",
+        title: "topic.share.help",
+        action: context => {
+          context.appEvents.trigger(
+            "share:url",
+            context.get("topic.shareUrl"),
+            $("#topic-footer-buttons")
+          );
+        },
+        dropdown: context => context.site.mobileView,
+        class: "share",
```

Comment:
I thought `class` was a reserved word? It might be safer to use classNames or similar.

Reply:
Yes I think that would be better, class is reserved in some cases only, but probably better to go with `classNames` I like it :+1: